src/test/groovy/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataFetcherSpec.groovy

```diff
@@ -4,28 +4,28 @@ import spock.lang.Specification
 
 class BloggersDataFetcherSpec extends Specification {
 
-    def "Should store empty optional when url is not valid"() {
+    def "Should not throw exception when url is not valid"() {
         given:
             String urlString = "invalid"
             String urlString2 = "invalid"
-        when:
             BloggersDataFetcher fetcher = new BloggersDataFetcher(urlString, urlString2, Stub(BloggersDataUpdater))
+        when:
+            fetcher.refreshData()
         then:
-            !fetcher.bloggersUrlOptional.isPresent()
-            !fetcher.companiesUrlOptional.isPresent()
+            notThrown Exception
     }
 
-    def "Should store valid URL with valid input"() {
+    def "Should update bloggers data from valid url"() {
         given:
-            String urlString = "http://google.com"
-            String urlString2 = "http://facebook.com"
+            String urlString = getClass().getResource("bloggers.json").toExternalForm()
```

Comment:
maybe test_bloggers.json and test_companies.json to avoid accidental mistakes with production files?